controllers/idler/idler_controller_test.go

```diff
@@ -41,6 +41,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	RestartCountWithinThresholdContainer1 = 30
+	RestartCountWithinThresholdContainer2 = 24
+	RestartCountOverThreshold             = 52
+	TestIdlerTimeOutSeconds               = 540
```

Comment:
I wanted to keep the `testIdlerTimeoutSeconds` to be different from `RequeueTimeThresholdSeconds`. Making it shorter was making the VMs idle too fast and assertions to fail at the count of pods being tracked. Keeping the `IdlerTimeoutSeconds` greater keeps the existing flow of tests with the max timeout of 5m.